python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -471,7 +475,22 @@ def resolve_runRequests(self, _graphene_info: ResolveInfo):
         if not self._run_requests:
             return self._run_requests
 
-        return [GrapheneRunRequest(run_request) for run_request in self._run_requests]
+        # filter out run requests that already have runs matching their run_keys
+        if isinstance(self._instigator, RemoteSensor):
+            existing_runs = fetch_existing_runs(
+                _graphene_info.context.instance,
+                self._instigator,
+                self._run_requests,
+            )
```

Comment:
we could also make this a separate runRequestsWithExistingRunKeys field - this would open the way for a frontend change that still showed them in the UI in some fashion, but explained why they were filtered out